strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactResolutionServiceImpl.java

```diff
@@ -37,15 +40,17 @@
     @Inject
     private RepositoryProviderRegistry repositoryProviderRegistry;
 
+    @Inject
+    private LayoutProviderRegistry layoutProviderRegistry;
 
     @Override
     public ArtifactInputStream getInputStream(String storageId,
                                               String repositoryId,
                                               String artifactPath)
-            throws IOException,
-                   NoSuchAlgorithmException,
-                   ArtifactTransportException,
-                   ProviderImplementationException
+        throws IOException,
```

Comment:
@sbespalov : Would you mind reverting the old indentation for the `throws` clause (here and across your change list)? Thanks! :)